### PR TITLE
New package: gdcalc-3.4

### PR DESCRIPTION
**Pull request by bhepple**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Review by Chocimier — COMMENTED** [review]
Btw, even if you need to make new git branch (or clone), you can reuse pull request, see [CONTRIBUTING.md#review](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#review)

File: srcpkgs/gdcalc/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'gdcalc'
+pkgname=gdcalc
+version=3.4
+revision=1
+archs="x86_64"
```
[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/gdcalc/template]: Please remove restriction. We want packages to build on all current and future archs. Please also squash commits.

**Comment by Chocimier** [comment]
I did recreate branch with single commit for you.

**Comment by bhepple** [comment]
Chocimier - thanks!

**Pull request merged**